django/apps/config.py

```diff
@@ -9,9 +9,7 @@
 
 
 class AppConfig(object):
-    """
-    Class representing a Django application and its configuration.
-    """
+    """Class representing a Django application and its configuration."""
```

Comment:
If there aren't any content changes required, don't bother changing a docstring like this.